Commands/CheckForbidded.ts

```diff
@@ -0,0 +1,28 @@
+import {Command} from '../Class/command';
+import low = require("lowdb");
+import FileSync = require("lowdb/adapters/FileSync");
+
+export class CheckForbidded extends Command {
+
+    static match(message) {
+        return !message.content.startsWith("$") && !message.author.bot;
```

Comment:
`       !message.author.bot;
` is enought

Reply:
If we check forbidded word in a message, without checking if it's the bot, the listforbid command to list all forbidded words will be automatically deleted

Reply:
If we check the forbidded words, without checking if it's a command, and if we use unforbid command to allow forbidded words, these command will be deleted besause it's contain forbidded words.